pkg/irreparable/irreparable.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparable
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/migrate"
+	dbx "storj.io/storj/pkg/irreparable/dbx"
+)
+
+var (
+	mon = monkit.Package()
+)
+
+// Database implements the irreparable RPC service
+type Database struct {
+	mu     sync.Mutex
+	DB     *dbx.DB
+	logger *zap.Logger
+}
+
+// RemoteSegmentInfo is info about a single entry stored in the irreparable db
+type RemoteSegmentInfo struct {
+	SegmentPath        []byte
+	SegmentDetail      []byte
+	LostPiecesCount    int64
+	RepairUnixSec      int64
+	RepairAttemptCount int64
+}
+
+// NewServer creates instance of Server
+func NewServer(source string, logger *zap.Logger) (*Database, error) {
```

Comment:
Rename `NewServer` to `New`. Because it's not a server, and since it's already in a package `irreparabledb` there's no need to mention `DB` in the constructor.

Reply:
Done